crux-core/src/crux/project.clj

```diff
@@ -0,0 +1,182 @@
+(ns ^:no-doc crux.project
```

Comment:
Not sure what a better name would be, but project.clj can be a bit confusing :-)

Reply:
`crux.eql.project` ?

Reply:
`crux.eql-project`